comm/sensorhub-comm-ble/src/main/java/org/sensorhub/api/comm/ble/IGattField.java

```diff
@@ -44,7 +44,7 @@ public interface IGattField
     public static final int PROPERTY_INDICATE = 32;
     public static final int PROPERTY_SIGNED_WRITE = 64;
     public static final int PROPERTY_EXTENDED_PROPS = 128;
-
+    public static final byte[] ENABLE_NOTIFICATION_VALUE = new byte[0];
```

Comment:
Why declare a byte array whose one and only value is it being initialized to a byte array whose "content" is always empty?